index.html

```diff
@@ -3666,8 +3666,8 @@ <h4>Not Mapped</h4>
 					<span class="property">Property: <code>AXRoleDescription</code>: <code>&lt;value&gt;</code></span>
 				</td>
 			</tr>
-			<tr id="ariaRoleDescriptionEmptyWhiteSpaceString">
-				<th><a class="property-reference" href="#aria-roledescription"><code>aria-roledescription</code></a> is empty or whitespace characters</th>
+			<tr id="ariaRoleDescriptionEmptyString">
+				<th><a class="property-reference" href="#aria-roledescription"><code>aria-roledescription</code></a> is the empty string</th>
```

Comment:
is "is the empty string" the same as "undefined"? Should we say "is undefined or the empty string"?

Reply:
good question.  i see in the ARIA spec the empty string and undefined are not necessarily treated as one thing. 

e.g., with `aria-current` 
> If the attribute is not present or its value is an empty string or `undefined`...

Reply:
> is "is the empty string" the same as "undefined"? 

No, but I think the difference does not matter in the context of `[aria-roledescription]`. It does for example with `img:not([alt])` vs `img[alt=""]`

> Should we say "is undefined or the empty string"?

IMO yes

Reply:
other than that, LGTM

Reply:
i've made the change.  this may need to spread out to the other PRs